vendor/modules.txt

```diff
@@ -541,7 +541,7 @@ github.com/hashicorp/hcl2/hclwrite
 github.com/hashicorp/hil/ast
 # github.com/hashicorp/logutils v1.0.0
 github.com/hashicorp/logutils
-# github.com/hashicorp/terraform v0.12.13
+# github.com/hashicorp/terraform v0.12.16
```

Comment:
Does `#` mean commented out?

Reply:
Yes. It's usually used to document the version of the installed dependency as above with "logutils.". But I think since they moved to using the `terraform-plugin-sdk` for managing the terraform dependencies instead of the builtin terraform apis, it doesn't install any.
I tried removing it all together, but `go mod` just adds it back.